ndoh_hub/urls.py

```diff
@@ -42,6 +44,8 @@
 v2router.register("postbirthregistrations", PostbirthRegistrationViewSet)
 v2router.register("researchoptins", ResearchOptinSwitchViewSet)
 v2router.register("messages", MessagesViewSet, basename="messages")
+v2router.register("eddeswitches", EddSwitchViewSet)
```

Comment:
Should this rather be `eddswitches`, instead of `eddeswitches`?

Reply:
Yes. Correcting that.